src/Hl7.Fhir.Specification.Tests/Source/ZipSourceTests.cs

```diff
@@ -30,7 +32,8 @@ public void ListSummariesIncludingSubdirectories()
         public void ListSummariesExcludingSubdirectories()
         {
             var zipfile = Path.Combine("TestData", "ResourcesInSubfolder.zip");
-            var zip = new ZipSource(zipfile, new DirectorySourceSettings() { IncludeSubDirectories = false });
+            var extractDir = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
+            var zip = new ZipSource(zipfile, extractDir, new DirectorySourceSettings() { IncludeSubDirectories = false });
```

Comment:
Why should we pass here the extract directory? There is still a constructor of `ZipSource` that generates a default extraction directory. Right?